analytics/analytics.js

```diff
@@ -25,18 +25,25 @@ if (window.trackingEnabled()) {
   gtag('js', new Date());
   gtag('config', 'UA-49796218-2');
 
-  window.addEventListener("DOMContentLoaded", () => {
-    // Decorate outbound links from the top frame
-    document.addEventListener("click", decoratedLinkHandler, true);
-  });
+  // Decorate outbound links from the top frame of whitelisted pages
+  const OUTBOUND_LINKS_WHITELIST = new Set([
+    '/',
+    '/index.html',
+  ]);
+
+  if (OUTBOUND_LINKS_WHITELIST.has(window.location.pathname)) {
+    window.addEventListener("DOMContentLoaded", () => {
+      document.addEventListener("click", decoratedLinkHandler, true);
+    });
+  }
 }
 
 function decoratedLinkHandler(e) {
   if (e.button != 0) {
     return;
   }
   let anchor = findAnchor(e.target);
-  if (anchor && anchor.hostname != window.location.hostname) {
+  if (anchor && anchor.href) {
```

Comment:
Can we just leave the `href` check out?
Worst-case we just get some more "(not set)" events.
On the plus side, we see when something goes wrong with the instrumentation (as we just did).